simplified-app-shared/src/main/java/org/nypl/simplified/app/MainSettingsAccountActivity.java

```diff
@@ -890,97 +904,101 @@ protected void onResume() {
     in_account_name_text.setText(MainSettingsAccountActivity.this.account.getName());
     in_account_subtitle_text.setText(MainSettingsAccountActivity.this.account.getSubtitle());
 
+
+    final BooksType userAccount;
+    if (this.account == null) {
+      userAccount = Simplified.getCatalogAppServices().getBooks();
+    } else {
+      userAccount = Simplified.getBooks(this.account, this, Simplified.getCatalogAppServices().getAdobeDRMExecutor());
+    }
+
+    final AccountsDatabaseType accounts_database  = Simplified.getAccountsDatabase(this.account, this);
+    if (!accounts_database.accountGetCredentials().isSome()) {
+      this.sync_table_row.setVisibility(View.GONE);
+      LOG.debug("No user currently signed in, bypassing UI update and Sync Status Initiation.");
+      return;
+    }
+
+    final AccountCredentials creds = ((Some<AccountCredentials>) accounts_database.accountGetCredentials()).get();
+
+    if (syncButtonShouldBeVisible()) {
+      if (this.annotationsManager == null) {
+        this.annotationsManager = new AnnotationsManager(this.account, creds, this);
+      }
+      this.sync_table_row.setVisibility(View.VISIBLE);
+      checkServerSyncPermission(userAccount);
+    } else {
+      this.sync_table_row.setVisibility(View.GONE);
+    }
+
+    in_table_with_code.setVisibility(View.VISIBLE);
+    in_table_signup.setVisibility(View.GONE);
+
+    in_barcode_text.setText(creds.getBarcode().toString());
+    in_barcode_text.setContentDescription(creds.getBarcode().toString().replaceAll(".(?=.)", "$0,"));
+    in_pin_text.setText(creds.getPin().toString());
+    in_pin_text.setContentDescription(creds.getPin().toString().replaceAll(".(?=.)", "$0,"));
+
+    in_eula_checkbox.setEnabled(false);
+
+    in_login.setText(rr.getString(R.string.settings_log_out));
+    in_login.setOnClickListener( v -> {
```

Comment:
what is `v`?

Reply:
In Java lambda syntax, I don't seem to see an equivalen "_" syntax to omit unused parameters. In this case the compile auto-completed the logout dialog "view" as just "v", but I don't mind editing those for clarity.